Scenes/Tavern/Buttons/TavernBtn.gd

```diff
@@ -14,9 +14,8 @@ export(PriceType) var price_type
 export(String) var popup_title 
 
 onready var game_data = get_tree().get_current_scene().find_node("GameData")
-
 onready var elf_stats = get_node("/root/World/ElfStats")
-
+onready var publician = get_tree().get_current_scene().find_node("Publician")
```

Comment:
I see small typo here. You added additional 'i' lettern in wariable name (and node name too). 
![obraz](https://user-images.githubusercontent.com/40212297/63762029-6da12a00-c8c2-11e9-9586-49685e650d0e.png)